spec/spec-helper.coffee

```diff
@@ -1,5 +1,12 @@
 require 'coffee-cache'
+Point = require '../src/point'
 
 exports.expectValues = (actual, expected) ->
   for expectedKey, expectedValue of expected
     expect(actual[expectedKey]).toEqual expectedValue
+
+exports.expectMappings = (mappings, {fromLayer, toLayer}) ->
```

Comment:
Nice point you bring up. To be honest, my Ruby (+ RSpec) background made me do this automatically, but it shouldn't probably behave that way.

Even though it's working for some reason, I will simply require `spec-helper` where needed until we understand what's going on there.

Thanks.